BugsnagPerformance/Assets/BugsnagPerformance/Scripts/Public/PerformanceConfiguration.cs

```diff
@@ -27,6 +27,59 @@ public PerformanceConfiguration(string apiKey)
         internal float PValueCheckIntervalSeconds = 30f;
 
         //Public config
+        private const int DEFAULT_ATTRIBUTE_STRING_VALUE_LIMIT = 1024;
+        private int _attributeStringValueLimit = DEFAULT_ATTRIBUTE_STRING_VALUE_LIMIT;
+        public int AttributeStringValueLimit
+        {
+            get => _attributeStringValueLimit;
+            set
+            {
+                if (value > 0 && value <= 10000)

```

Comment:
Seeing as we have constants for the default, seems like it would be nicely readable if we defined pairs - e.g. `ATTRIBUTE_STRING_VALUE_LIMIT_DEFAULT` and `ATTRIBUTE_STRING_VALUE_LIMIT_MAX`

Reply:
... also placing all the consts at the top of the file